src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -46,13 +47,17 @@ public function renderBlock(FormView $view, $resource, string $blockName, array
 
         $context = $this->environment->mergeGlobals($variables);
 
-        ob_start();
-
         // By contract,This method can only be called after getting the resource
         // (which is passed to the method). Getting a resource for the first time
         // (with an empty cache) is guaranteed to invoke loadResourcesFromTheme(),
         // where the property $template is initialized.
 
+        if ($this->template instanceof YieldingTemplate) {
```

Comment:
Do we actually still need to use the internal class `Template` in `$this->template`, forcing us to use more internal classes there ?
Couldn't we instead rely on the public API of `TemplateWrapper`, which has a `renderBlock` method (and btw, meging globals would also be useless as Twig would be doing it in its public entry points)